Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -204,15 +212,24 @@ def create_pull_request(cls, repository, args, branch_point):
                 labels.remove(cls.BLOCKED_LABEL)
                 pr_issue.set_labels([])
 
-        target = 'fork' if isinstance(remote_repo, remote.GitHub) else source_remote
+        if isinstance(remote_repo, remote.GitHub):
+            target = 'fork' if source_remote == 'origin' else '{}-fork'.format(source_remote)
+            if not repository.config().get('remote.{}.url'.format(target)):
+                sys.stderr.write("'{}' is not a remote in this repository. Have you run `{} setup` yet?\n".format(
```

Comment:
One side note, I think we are missing setup parts to add a customized remote ? And the user input remote may not support fork

Reply:
We are handling this in setup after https://commits.webkit.org/249702@main